Dockerfile

```diff
@@ -3,13 +3,11 @@ FROM registry.opensource.zalan.do/stups/ubuntu:latest
 #making this a cachable point as compile takes forever without -j
 
 RUN apt-get update && apt-get -y install python-pip python-dev libev4 libev-dev python-psycopg2 libpq-dev libldap2-dev libsasl2-dev libssl-dev libsnappy-dev iputils-ping freetds-dev git && \
-    pip2 install -U pip setuptools urllib3 Cython
+    pip2 install -U pip setuptools urllib3 Cython opentracing-utils
```

Comment:
May be we can move ``opentracing-utils`` to ``requirements.txt`` with the rest of "normal" packages.